[18Mar2025 18:44:52.021] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeserver, --fml.forgeVersion, 47.3.7, --fml.mcVersion, 1.20.1, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20230612.114412, nogui]
[18Mar2025 18:44:52.025] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 10.0.9+10.0.9+main.dcd20f30 starting: java version 17.0.14 by Ubuntu; OS Linux arch amd64 version 6.8.0-1021-azure
[18Mar2025 18:44:54.735] [main/INFO] [net.minecraftforge.fml.loading.ImmediateWindowHandler/]: ImmediateWindowProvider not loading because launch target is forgeserver
[18Mar2025 18:44:54.907] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/workspaces/serber1/servidor_minecraft/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2365!/ Service=ModLauncher Env=SERVER
[18Mar2025 18:44:55.303] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file /workspaces/serber1/servidor_minecraft/libraries/net/minecraftforge/fmlcore/1.20.1-47.3.7/fmlcore-1.20.1-47.3.7.jar is missing mods.toml file
[18Mar2025 18:44:55.308] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file /workspaces/serber1/servidor_minecraft/libraries/net/minecraftforge/javafmllanguage/1.20.1-47.3.7/javafmllanguage-1.20.1-47.3.7.jar is missing mods.toml file
[18Mar2025 18:44:55.310] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file /workspaces/serber1/servidor_minecraft/libraries/net/minecraftforge/lowcodelanguage/1.20.1-47.3.7/lowcodelanguage-1.20.1-47.3.7.jar is missing mods.toml file
[18Mar2025 18:44:55.311] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file /workspaces/serber1/servidor_minecraft/libraries/net/minecraftforge/mclanguage/1.20.1-47.3.7/mclanguage-1.20.1-47.3.7.jar is missing mods.toml file
[18Mar2025 18:44:55.639] [main/WARN] [net.minecraftforge.jarjar.selection.JarSelector/]: Attempted to select two dependency jars from JarJar which have the same identification: Mod File:  and Mod File: . Using Mod File: 
[18Mar2025 18:44:55.641] [main/WARN] [net.minecraftforge.jarjar.selection.JarSelector/]: Attempted to select a dependency jar for JarJar which was passed in as source: playeranimator. Using Mod File: /workspaces/serber1/servidor_minecraft/mods/player-animation-lib-forge-1.0.2-rc1+1.20.jar
[18Mar2025 18:44:55.641] [main/INFO] [net.minecraftforge.fml.loading.moddiscovery.JarInJarDependencyLocator/]: Found 4 dependencies adding them to mods collection
[18Mar2025 18:44:59.157] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
[18Mar2025 18:44:59.178] [main/ERROR] [mixin/]: Mixin config mixins.recrafted_creatures.json does not specify "minVersion" property
[18Mar2025 18:44:59.184] [main/ERROR] [mixin/]: Mixin config mixins.duclib.json does not specify "minVersion" property
[18Mar2025 18:44:59.372] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [com.sonicether.soundphysics.MixinConnector]
[18Mar2025 18:44:59.376] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeserver' with arguments [nogui]
[18Mar2025 18:44:59.562] [main/INFO] [ModernFix/]: Loaded configuration file for ModernFix 5.20.2+mc1.20.1: 87 options available, 0 override(s) found
[18Mar2025 18:44:59.563] [main/INFO] [ModernFix/]: Applying Nashorn fix
[18Mar2025 18:44:59.583] [main/INFO] [ModernFix/]: Applied Forge config corruption patch
[18Mar2025 18:45:02.665] [main/INFO] [MixinExtras|Service/]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[18Mar2025 18:45:03.580] [main/INFO] [net.minecraft.server.Bootstrap/]: ModernFix reached bootstrap stage (17.34 s after launch)
[18Mar2025 18:45:03.850] [main/WARN] [mixin/]: @Final field delegatesByName:Ljava/util/Map; in modernfix-forge.mixins.json:perf.forge_registry_alloc.ForgeRegistryMixin should be final
[18Mar2025 18:45:03.850] [main/WARN] [mixin/]: @Final field delegatesByValue:Ljava/util/Map; in modernfix-forge.mixins.json:perf.forge_registry_alloc.ForgeRegistryMixin should be final
[18Mar2025 18:45:06.484] [main/INFO] [net.minecraft.server.Bootstrap/]: Vanilla bootstrap took 2899 milliseconds
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:pufferfish_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:salmon_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:cod_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:tropical_fish_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:axolotl_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.487] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:powder_snow_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.488] [modloading-worker-0/INFO] [Bookshelf/]: Fixing MC-151457. Crafting remainder for minecraft:tadpole_bucket is now minecraft:bucket.
[18Mar2025 18:45:11.531] [modloading-worker-0/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 47.3.7, for MC 1.20.1 with MCP 20230612.114412
[18Mar2025 18:45:11.532] [modloading-worker-0/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v47.3.7 Initialized
[18Mar2025 18:45:11.570] [modloading-worker-0/INFO] [de.keksuccino.konkrete.Konkrete/]: [KONKRETE] Successfully initialized!
[18Mar2025 18:45:11.570] [modloading-worker-0/INFO] [de.keksuccino.konkrete.Konkrete/]: [KONKRETE] Server-side libs ready to use!
[18Mar2025 18:45:12.008] [modloading-worker-0/INFO] [dev.architectury.networking.forge.NetworkManagerImpl/]: Registering C2S receiver with id architectury:sync_ids
[18Mar2025 18:45:12.327] [modloading-worker-0/INFO] [dev.architectury.networking.forge.NetworkManagerImpl/]: Registering C2S receiver with id magistuarmory:packet_long_reach_attack
[18Mar2025 18:45:12.331] [modloading-worker-0/INFO] [dev.architectury.networking.forge.NetworkManagerImpl/]: Registering C2S receiver with id magistuarmory:packet_lance_collision
[18Mar2025 18:45:12.396] [modloading-worker-0/INFO] [com.cupboard.Cupboard/]: Loaded config for: cupboard.json
[18Mar2025 18:45:12.417] [modloading-worker-0/WARN] [de.keksuccino.justzoom.JustZoom/]: [JUST ZOOM] Disabling 'Just Zoom' since it's a client-side mod and current environment is server!
[18Mar2025 18:45:14.789] [main/INFO] [ModernFix/]: Bypassed Mojang DFU
[18Mar2025 18:45:14.790] [main/INFO] [ModernFix/]: Instantiating Mojang DFU
[18Mar2025 18:45:15.700] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class com.modderg.tameablebeasts.server.entity.TBAnimal from interface com.modderg.tameablebeasts.server.entity.TBRideable
[18Mar2025 18:45:15.701] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class com.modderg.tameablebeasts.server.entity.TBAnimal from class com.modderg.tameablebeasts.server.entity.custom.GroundBeetleEntity
[18Mar2025 18:45:15.788] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.788] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.788] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.790] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.791] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.791] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.791] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.791] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:15.793] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.794] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.794] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.795] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.796] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.798] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.798] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.799] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.799] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.799] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.800] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.800] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:15.802] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.807] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.807] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:15.808] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.808] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:15.808] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:15.808] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.809] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.809] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.809] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.809] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.811] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.817] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.817] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.818] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.818] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.820] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.VillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:45:15.821] [main/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.GrimReaperEntity from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:45:16.349] [modloading-worker-0/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loading Open Parties and Claims!
[18Mar2025 18:45:16.353] [modloading-worker-0/INFO] [com.connectivity.Connectivity/]: Connectivity initialized
[18Mar2025 18:45:16.448] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Compatibility version 18
[18Mar2025 18:45:16.548] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Loading plugins
[18Mar2025 18:45:16.560] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Loaded 1 plugin(s)
[18Mar2025 18:45:16.562] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Initializing plugins
[18Mar2025 18:45:16.580] [modloading-worker-0/INFO] [Sound Physics - General/]: Initializing Simple Voice Chat integration
[18Mar2025 18:45:16.580] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Initialized 1 plugin(s)
[18Mar2025 18:45:16.581] [modloading-worker-0/INFO] [voicechat/]: [voicechat] Registering events for 'sound_physics_remastered'
[18Mar2025 18:45:16.721] [Thread-0/INFO] [ModernFix/]: Please use /mfsrc to reload any changed mod config files
[18Mar2025 18:45:16.828] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [clumps] Starting version check at https://updates.blamejared.com/get?n=clumps&gv=1.20.1
[18Mar2025 18:45:17.186] [modloading-worker-0/INFO] [xaero.map.server.WorldMapServer/]: Loading Xaero's World Map - Stage 1/2 (Server)
[18Mar2025 18:45:17.187] [modloading-worker-0/INFO] [xaero.minimap.XaeroMinimap/]: Loading Xaero's Minimap - Stage 1/2 (Server)
[18Mar2025 18:45:17.211] [main/INFO] [xaero.map.server.WorldMapServer/]: Loading Xaero's World Map - Stage 2/2 (Server)
[18Mar2025 18:45:17.211] [main/INFO] [xaero.minimap.XaeroMinimap/]: Loading Xaero's Minimap - Stage 2/2 (Server)
[18Mar2025 18:45:17.518] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [clumps] Found status: BETA Current: 12.0.0.4 Target: 12.0.0.4
[18Mar2025 18:45:17.518] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [bookshelf] Starting version check at https://updates.blamejared.com/get?n=bookshelf&gv=1.20.1
[18Mar2025 18:45:17.665] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [bookshelf] Found status: BETA Current: 20.2.13 Target: 20.2.13
[18Mar2025 18:45:17.666] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/net/minecraftforge/forge/promotions_slim.json
[18Mar2025 18:45:17.924] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: OUTDATED Current: 47.3.7 Target: 47.4.0
[18Mar2025 18:45:17.925] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [voicechat] Starting version check at https://update.maxhenkel.de/forge/voicechat
[18Mar2025 18:45:18.105] [main/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[18Mar2025 18:45:18.577] [main/WARN] [net.minecraft.server.packs.VanillaPackResourcesBuilder/]: Assets URL 'union:/workspaces/serber1/servidor_minecraft/libraries/net/minecraft/server/1.20.1-20230612.114412/server-1.20.1-20230612.114412-srg.jar%23213!/assets/.mcassetsroot' uses unexpected schema
[18Mar2025 18:45:18.577] [main/WARN] [net.minecraft.server.packs.VanillaPackResourcesBuilder/]: Assets URL 'union:/workspaces/serber1/servidor_minecraft/libraries/net/minecraft/server/1.20.1-20230612.114412/server-1.20.1-20230612.114412-srg.jar%23213!/data/.mcassetsroot' uses unexpected schema
[18Mar2025 18:45:18.722] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [voicechat] Found status: OUTDATED Current: 1.20.1-2.5.26 Target: 1.20.1-2.5.28
[18Mar2025 18:45:18.724] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [sound_physics_remastered] Starting version check at https://update.maxhenkel.de/forge/sound_physics_remastered
[18Mar2025 18:45:18.874] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [sound_physics_remastered] Found status: AHEAD Current: 1.20.1-1.4.8 Target: null
[18Mar2025 18:45:18.874] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [bonfires] Starting version check at https://raw.githubusercontent.com/Wehavecookies56/Bonfires/master/update.json
[18Mar2025 18:45:19.076] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [bonfires] Found status: BETA Current: 1.2.18 Target: 1.2.17
[18Mar2025 18:45:20.724] [main/INFO] [com.github.alexthe666.alexsmobs.AlexsMobs/]: Adding datapack listener capsid_recipes
[18Mar2025 18:45:20.931] [Worker-Main-3/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag magistuarmory:darkening_templates as it is missing following references: magistuarmoryaddon:darkening_template (from [1.20.x-forge]-Epic-Knights-Addon-1.22.jar)
[18Mar2025 18:45:20.945] [Worker-Main-3/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag magistuarmory:gilding_templates as it is missing following references: magistuarmoryaddon:gilding_template (from [1.20.x-forge]-Epic-Knights-Addon-1.22.jar)
[18Mar2025 18:45:22.050] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[1].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.050] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[0].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.050] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[9].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.050] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[8].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.050] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[2].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.051] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[7].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.051] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[3].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.051] [main/WARN] [net.minecraft.world.level.storage.loot.LootDataManager/]: Found loot table element validation problem in {loot_tables:mimic:entities/mimic}.pools[0].entries[4].functions[1]: Parameters [<parameter minecraft:tool>] are not provided in this context
[18Mar2025 18:45:22.070] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe antiquelegacy:mars_of_todi_thorax: com.google.gson.JsonSyntaxException: Unknown item 'antiquelegacy:mars_of_todi_thorax'
[18Mar2025 18:45:22.079] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe antiquelegacy:furnace/gold_nugget_blasting: com.google.gson.JsonSyntaxException: Unknown item 'antiquelegacy:golden_laurel_wreath'
[18Mar2025 18:45:22.103] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe magistuarmoryaddon:darkblued_sallet_from_dark_gothic_helmet_darkening: com.google.gson.JsonSyntaxException: Invalid or unsupported recipe type 'minecraft:smithing'
[18Mar2025 18:45:22.104] [main/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe recrafted_creatures:creature_catalogue as it's serializer returned null
[18Mar2025 18:45:22.112] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe antiquelegacy:furnace/bronze_nugget_blasting: com.google.gson.JsonSyntaxException: Unknown item 'antiquelegacy:left_gladiator_shoulder_pad'
[18Mar2025 18:45:22.115] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe magistuarmoryaddon:darkblued_armet_from_dark_knight_helmet_darkening: com.google.gson.JsonSyntaxException: Invalid or unsupported recipe type 'minecraft:smithing'
[18Mar2025 18:45:22.124] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe epicknights_open_helms:glided_dark_clos: com.google.gson.JsonSyntaxException: Unknown item 'epicknights_open_helms:glided_dark_close_helmet'
[18Mar2025 18:45:22.133] [main/ERROR] [net.minecraft.world.item.crafting.RecipeManager/]: Parsing error loading recipe antiquelegacy:furnace/iron_nugget_blasting: com.google.gson.JsonSyntaxException: Unknown item 'antiquelegacy:iron_left_hand_manika'
[18Mar2025 18:45:22.167] [main/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Loaded 13 recipes
[18Mar2025 18:45:22.368] [main/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 1845 advancements
[18Mar2025 18:45:22.373] [main/INFO] [rechiseled/]: Loaded 79 chiseling recipes
[18Mar2025 18:45:22.771] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting minecraft server version 1.20.1
[18Mar2025 18:45:22.772] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Loading properties
[18Mar2025 18:45:22.772] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Default game type: SURVIVAL
[18Mar2025 18:45:22.772] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Generating keypair
[18Mar2025 18:45:22.882] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting Minecraft server on *:25565
[18Mar2025 18:45:22.930] [Server thread/INFO] [net.minecraft.server.network.ServerConnectionListener/]: Using epoll channel type
[18Mar2025 18:45:23.033] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[18Mar2025 18:45:23.033] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: The server will make no attempt to authenticate usernames. Beware.
[18Mar2025 18:45:23.034] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[18Mar2025 18:45:23.035] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: To change this, set "online-mode" to "true" in the server.properties file.
[18Mar2025 18:45:23.176] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Initializing Open Parties and Claims for the server...
[18Mar2025 18:45:23.194] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loading server info!
[18Mar2025 18:45:23.273] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loading player configs...
[18Mar2025 18:45:23.320] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loaded player configs!
[18Mar2025 18:45:23.320] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loading parties...
[18Mar2025 18:45:23.322] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loaded parties!
[18Mar2025 18:45:23.323] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Fixing party inconsistencies...
[18Mar2025 18:45:23.366] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Registered party system for OPAC: default
[18Mar2025 18:45:23.367] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Registered permission system for OPAC: permission_api
[18Mar2025 18:45:23.379] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Preparing level "world"
[18Mar2025 18:45:23.857] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Preparing start region for dimension minecraft:overworld
[18Mar2025 18:45:25.197] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:25.197] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:25.198] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:25.365] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:25.884] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:26.364] [Worker-Main-1/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[18Mar2025 18:45:26.682] [Server thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Time elapsed: 2824 ms
[18Mar2025 18:45:26.683] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Done (3.629s)! For help, type "help"
[18Mar2025 18:45:26.770] [Server thread/INFO] [net.minecraftforge.server.permission.PermissionAPI/]: Successfully initialized permission handler forge:default_handler
[18Mar2025 18:45:26.773] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loading claims...
[18Mar2025 18:45:26.776] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Loaded claims in 3ms
[18Mar2025 18:45:26.777] [Server thread/WARN] [xaero.pac.OpenPartiesAndClaims/]: The configured permission system "prometheus" isn't registered!
[18Mar2025 18:45:26.777] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Configured OPAC to use the following permission system: permission_api
[18Mar2025 18:45:26.777] [Server thread/WARN] [xaero.pac.OpenPartiesAndClaims/]: The configured primary party system "argonauts_guilds" isn't registered!
[18Mar2025 18:45:26.778] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Configured OPAC to use the following party system as primary: default
[18Mar2025 18:45:26.782] [Server thread/INFO] [com.cupboard.Cupboard/]: Loaded config for: connectivity.json
[18Mar2025 18:45:26.783] [Server thread/WARN] [voicechat/]: [voicechat] Running in offline mode - Voice chat encryption is not secure!
[18Mar2025 18:45:26.792] [VoiceChatServerThread/INFO] [voicechat/]: [voicechat] Voice host is 'feed-see.gl.at.ply.gg:1081'
[18Mar2025 18:45:26.793] [Server thread/WARN] [ModernFix/]: Dedicated server took 40.559 seconds to load
[18Mar2025 18:45:26.793] [VoiceChatServerThread/INFO] [voicechat/]: [voicechat] Voice chat server started at port 1081
[18Mar2025 18:47:36.656] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class net.minecraft.world.entity.player.Player from class tschipp.carryon.common.carry.CarryOnDataManager
[18Mar2025 18:47:36.716] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: PlagaXNZ[/127.71.30.2:41559] logged in with entity id 109 at (360.994900901392, 79.0, 813.2545342849058)
[18Mar2025 18:47:36.794] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: PlagaXNZ joined the game
[18Mar2025 18:47:36.843] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Updating all forceload tickets for 116e9c76-7157-3f7d-8106-557b209a5b07
[18Mar2025 18:47:37.534] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.535] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.536] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.536] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.536] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.537] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.537] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.537] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:47:37.538] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.538] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.538] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.539] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.539] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.539] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.540] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.540] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.540] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.541] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.541] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.541] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:47:37.542] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.542] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.542] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:47:37.543] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.543] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:47:37.543] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CEnumParameter
[18Mar2025 18:47:37.544] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.544] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.544] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.544] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.545] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.545] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.545] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.545] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.546] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.546] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.546] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class forge.net.mca.entity.ZombieVillagerEntityMCA from class forge.net.mca.util.network.datasync.CDataParameter
[18Mar2025 18:47:37.564] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of PlagaXNZ (18)
[18Mar2025 18:47:37.566] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to PlagaXNZ
[18Mar2025 18:47:39.361] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 116e9c76-7157-3f7d-8106-557b209a5b07
[18Mar2025 18:47:40.357] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 116e9c76-7157-3f7d-8106-557b209a5b07
[18Mar2025 18:47:40.362] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player PlagaXNZ (116e9c76-7157-3f7d-8106-557b209a5b07) successfully connected to voice chat
[18Mar2025 18:52:23.832] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Made PlagaXNZ a server operator
[18Mar2025 18:52:56.353] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: PlagaXNZ was slain by Enderman
[18Mar2025 18:56:56.177] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [PlagaXNZ: Created team [fas]]
[18Mar2025 18:57:10.619] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [PlagaXNZ: Added PlagaXNZ to team [fas]]
[18Mar2025 18:57:42.210] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [PlagaXNZ: Removed PlagaXNZ from any team]
[18Mar2025 19:00:08.078] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: Jukonso[/127.113.44.68:57875] logged in with entity id 3462 at (390.3014472769231, 49.0, 649.1188726522792)
[18Mar2025 19:00:08.087] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso joined the game
[18Mar2025 19:00:08.094] [Server thread/INFO] [xaero.pac.OpenPartiesAndClaims/]: Updating all forceload tickets for cdc82f89-27df-3e96-b744-2e7cd481dc97
[18Mar2025 19:00:11.067] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of Jukonso (18)
[18Mar2025 19:00:11.068] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to Jukonso
[18Mar2025 19:00:18.953] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player cdc82f89-27df-3e96-b744-2e7cd481dc97
[18Mar2025 19:00:19.957] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player cdc82f89-27df-3e96-b744-2e7cd481dc97
[18Mar2025 19:00:19.958] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player Jukonso (cdc82f89-27df-3e96-b744-2e7cd481dc97) successfully connected to voice chat
[18Mar2025 19:00:24.473] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2029ms or 40 ticks behind
[18Mar2025 19:01:02.583] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player cdc82f89-27df-3e96-b744-2e7cd481dc97 timed out
[18Mar2025 19:01:02.584] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player Jukonso
[18Mar2025 19:01:02.584] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to Jukonso
[18Mar2025 19:01:02.845] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player cdc82f89-27df-3e96-b744-2e7cd481dc97
[18Mar2025 19:01:03.845] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player cdc82f89-27df-3e96-b744-2e7cd481dc97
[18Mar2025 19:01:03.846] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player Jukonso (cdc82f89-27df-3e96-b744-2e7cd481dc97) successfully connected to voice chat
[18Mar2025 19:01:54.145] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2001ms or 40 ticks behind
[18Mar2025 19:02:13.125] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> donde andas??
[18Mar2025 19:02:37.303] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:37.311] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=925, z=1073200}"
}
[18Mar2025 19:02:37.312] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:37.313] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[94, 79, -22, 117, -83, -24, 29, 43, 2, 99, 27]"
]
[18Mar2025 19:02:39.090] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:39.091] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=1015, z=1073200}"
}
[18Mar2025 19:02:39.091] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:39.092] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-58, -69, 109, -41, 58, -124, -105, -118, -63, -24, -55]"
]
[18Mar2025 19:02:39.573] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:39.574] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=958, z=1073200}"
}
[18Mar2025 19:02:39.574] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:39.574] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[85, -101, 87, 64, 98, 13, 12, 1, -68, 84, -115]"
]
[18Mar2025 19:02:40.074] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> 642 1437
[18Mar2025 19:02:40.106] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.107] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=769, z=1073200}"
}
[18Mar2025 19:02:40.107] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 61 extra bytes: 
[18Mar2025 19:02:40.108] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "5a!*TNFBQL<d-1$ i.F6[\\Gy'0T:T"
]
[18Mar2025 19:02:40.290] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.291] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=805, z=1073200}"
}
[18Mar2025 19:02:40.291] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:40.292] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[59, -50, -90, -86, 125, -47, 5, -36, -118, 17, -24]"
]
[18Mar2025 19:02:40.585] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.586] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=1016, z=1073200}"
}
[18Mar2025 19:02:40.586] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:40.586] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-49, 105, -11, 68, 4, 23, -61, -10, -124, -71, -66]"
]
[18Mar2025 19:02:40.700] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.701] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=773, z=1073200}"
}
[18Mar2025 19:02:40.701] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 98 extra bytes: 
[18Mar2025 19:02:40.701] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "]|&l;PH(F9I3t i|HpN9$-@|z(q&+/,0/5"
]
[18Mar2025 19:02:40.702] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundPlayerInputPacket  id 31 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.703] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_134339_": 1.3641817E-15,
  "f_134340_": 0.0031877924,
  "f_134341_": false,
  "f_134342_": false
}
[18Mar2025 19:02:40.703] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 31 data of 109 extra bytes: 
[18Mar2025 19:02:40.703] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "F9I3t i|HpN9$-@|z(q&+/,0/527.ip.gl.ply.gg"
]
[18Mar2025 19:02:40.703] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundPongPacket  id 32 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:40.704] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_179721_": -1866545303
}
[18Mar2025 19:02:40.704] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 32 data of 130 extra bytes: 
[18Mar2025 19:02:40.704] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "|HpN9$-@|z(q&+/,0/527.ip.gl.ply.gg"
]
[18Mar2025 19:02:41.232] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.233] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=768, z=1073200}"
}
[18Mar2025 19:02:41.233] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 54 extra bytes: 
[18Mar2025 19:02:41.233] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "hriElJ]Z*\\ \"/S M\\_Q\\"
]
[18Mar2025 19:02:41.233] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundContainerClosePacket  id 12 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.234] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133967_": -117
}
[18Mar2025 19:02:41.234] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 12 data of 86 extra bytes: 
[18Mar2025 19:02:41.234] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "riElJ]Z*\\ \"/S M\\_Q\\#\\&+/,0/"
]
[18Mar2025 19:02:41.234] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundContainerButtonClickPacket  id 10 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.235] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133923_": -118,
  "f_133924_": 42
}
[18Mar2025 19:02:41.235] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 10 data of 159 extra bytes: 
[18Mar2025 19:02:41.235] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "\\ \"/S M\\_Q\\#\\&+/,0/27.ip.gl.ply.gg"
]
[18Mar2025 19:02:41.714] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.714] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=961, z=1073200}"
}
[18Mar2025 19:02:41.714] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 92 extra bytes: 
[18Mar2025 19:02:41.715] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "h@|Q'Pn\\ 4\"ieZVn5^6P&+/,0/5"
]
[18Mar2025 19:02:41.715] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundPlayerAbilitiesPacket  id 28 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.715] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_134254_": true
}
[18Mar2025 19:02:41.716] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 28 data of 98 extra bytes: 
[18Mar2025 19:02:41.716] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "h@|Q'Pn\\ 4\"ieZVn5^6P&+/,0/5"
]
[18Mar2025 19:02:41.717] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.717] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 28
}
[18Mar2025 19:02:41.717] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 42 extra bytes: 
[18Mar2025 19:02:41.717] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "/527.ip.gl.ply.gg"
]
[18Mar2025 19:02:41.718] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.718] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 53
}
[18Mar2025 19:02:41.718] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 37 extra bytes: 
[18Mar2025 19:02:41.718] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-64, 18, 19, 1, 19, 2, 19, 3, 1, 0, 0, -105, 0, 18, 0, 0, 15, 50, 55, 46, 105, 112, 46, 103, 108, 46, 112, 108, 121, 46, 103, 103, 0, 5, 1, 0, 0]"
]
[18Mar2025 19:02:41.718] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundLockDifficultyPacket  id 19 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.719] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_134105_": true
}
[18Mar2025 19:02:41.719] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 19 data of 43 extra bytes: 
[18Mar2025 19:02:41.719] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[19, 2, 19, 3, 1, 0, 0, -105, 0, 18, 0, 0, 15, 50, 55, 46, 105, 112, 46, 103, 108, 46, 112, 108, 121, 46, 103, 103, 0, 5, 1, 0, 0, 0, 10, 0, 8, 0, 29, 0, 23, 0, 24]"
]
[18Mar2025 19:02:41.719] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundLockDifficultyPacket  id 19 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.720] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_134105_": true
}
[18Mar2025 19:02:41.720] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 19 data of 66 extra bytes: 
[18Mar2025 19:02:41.720] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[19, 3, 1, 0, 0, -105, 0, 18, 0, 0, 15, 50, 55, 46, 105, 112, 46, 103, 108, 46, 112, 108, 121, 46, 103, 103, 0, 5, 1, 0, 0, 0, 10, 0, 8, 0, 29, 0, 23, 0, 24, 0, 11, 0, 2, 1, 0, 0, 13, 0, 26, 0, 24, 8, 4, 4, 3, 8, 7, 8, 5, 8, 6, 4, 1, 5]"
]
[18Mar2025 19:02:41.720] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundLockDifficultyPacket  id 19 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.721] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_134105_": true
}
[18Mar2025 19:02:41.721] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 19 data of 65 extra bytes: 
[18Mar2025 19:02:41.721] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[1, 0, 0, -105, 0, 18, 0, 0, 15, 50, 55, 46, 105, 112, 46, 103, 108, 46, 112, 108, 121, 46, 103, 103, 0, 5, 1, 0, 0, 0, 10, 0, 8, 0, 29, 0, 23, 0, 24, 0, 11, 0, 2, 1, 0, 0, 13, 0, 26, 0, 24, 8, 4, 4, 3, 8, 7, 8, 5, 8, 6, 4, 1, 5, 6]"
]
[18Mar2025 19:02:41.721] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.722] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=154624, y=-206, z=18874368}"
}
[18Mar2025 19:02:41.722] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 56 extra bytes: 
[18Mar2025 19:02:41.722] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[55, 46, 105, 112, 46, 103, 108, 46, 112, 108, 121, 46, 103, 103, 0, 5, 1, 0, 0, 0, 10, 0, 8, 0, 29, 0, 23, 0, 24, 0, 11, 0, 2, 1, 0, 0, 13, 0, 26, 0, 24, 8, 4, 4, 3, 8, 7, 8, 5, 8, 6, 4, 1, 5, 6, 5]"
]
[18Mar2025 19:02:41.723] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundChangeDifficultyPacket  id 2 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.723] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133814_": "HARD"
}
[18Mar2025 19:02:41.723] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 2 data of 2 extra bytes: 
[18Mar2025 19:02:41.723] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[1, 0]"
]
[18Mar2025 19:02:41.724] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.724] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 0
}
[18Mar2025 19:02:41.724] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 21 extra bytes: 
[18Mar2025 19:02:41.724] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[0, 18, 0, 0, 0, 43, 0, 9, 8, 3, 4, 3, 3, 3, 2, 3, 1, 0, 51, 0, 38]"
]
[18Mar2025 19:02:41.725] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.725] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 18
}
[18Mar2025 19:02:41.725] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 19 extra bytes: 
[18Mar2025 19:02:41.725] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[0, 0, 0, 43, 0, 9, 8, 3, 4, 3, 3, 3, 2, 3, 1, 0, 51, 0, 38]"
]
[18Mar2025 19:02:41.726] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.726] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 0
}
[18Mar2025 19:02:41.726] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 53 extra bytes: 
[18Mar2025 19:02:41.727] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "+3& u5b+noX4}e5dyp"
]
[18Mar2025 19:02:41.727] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:41.727] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 43
}
[18Mar2025 19:02:41.727] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 51 extra bytes: 
[18Mar2025 19:02:41.727] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "3& u5b+noX4}e5dyp"
]
[18Mar2025 19:02:42.090] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:42.091] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=1018, z=1073200}"
}
[18Mar2025 19:02:42.091] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 38 extra bytes: 
[18Mar2025 19:02:42.092] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "Tz*Io\\|7NU3z$= ,5$"
]
[18Mar2025 19:02:47.574] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> ando buscando aldea o algo
[18Mar2025 19:02:54.676] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:54.677] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=5768192, y=872, z=1122352}"
}
[18Mar2025 19:02:54.677] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:54.677] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[63, -92, 53, -38, -1, 4, -100, 84, -97, 66, 89]"
]
[18Mar2025 19:02:55.221] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[18Mar2025 19:02:55.222] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-8125440, y=960, z=905264}"
}
[18Mar2025 19:02:55.222] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[18Mar2025 19:02:55.222] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[53, 48, 25, 51, 68, 83, -56, -58, -19, -52, -104]"
]
[18Mar2025 19:03:04.024] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Jukonso moved wrongly!
[18Mar2025 19:03:54.581] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> 375 -1 594
[18Mar2025 19:04:12.417] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso was slain by Zombie
[18Mar2025 19:05:00.002] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2008ms or 40 ticks behind
[18Mar2025 19:05:26.648] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2004ms or 40 ticks behind
[18Mar2025 19:05:44.519] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2875ms or 57 ticks behind
[18Mar2025 19:06:02.198] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2704ms or 54 ticks behind
[18Mar2025 19:06:33.255] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> amigo q puta mierda era eso
[18Mar2025 19:06:37.941] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> osea WTF
[18Mar2025 19:06:44.125] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso was doomed to fall by Murmur
[18Mar2025 19:06:44.839] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> que cosa
[18Mar2025 19:06:47.870] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.870] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=61, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.870] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=63, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.870] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=61, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=63, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=61, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=63, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.871] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=61, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=63, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=63, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=416, y=62, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.872] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=416, y=63, z=2560}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=416, y=62, z=2560}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=416, y=63, z=2561}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=416, y=62, z=2561}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=61, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.873] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=63, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=63, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=63, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=63, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=63, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.874] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=417, y=62, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=417, y=63, z=2560}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=417, y=62, z=2560}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=417, y=63, z=2561}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 160], pos: BlockPos{x=417, y=62, z=2561}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=63, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=63, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2556}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.875] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=63, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2557}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=63, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2558}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=63, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=418, y=62, z=2559}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=419, y=62, z=2554}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=419, y=63, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:06:47.876] [Worker-Main-31/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [26, 159], pos: BlockPos{x=419, y=62, z=2555}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / recrafted_creatures:beaver_dam]
[18Mar2025 19:07:02.471] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso fell from a high place
[18Mar2025 19:07:15.166] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso fell from a high place
[18Mar2025 19:07:27.982] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> queres que te haga tp?
[18Mar2025 19:07:45.591] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Jukonso was blown up by Creeper
[18Mar2025 19:07:48.725] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> AMIGO ENCONTRE UN BOSS
[18Mar2025 19:08:07.179] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> si gil haceme tp a las coordenasdas q te envio
[18Mar2025 19:08:21.328] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> 375 -1 594
[18Mar2025 19:08:26.907] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> porfavor
[18Mar2025 19:08:32.108] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <PlagaXNZ> No hay qye abusar del tp
[18Mar2025 19:13:22.318] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Not Secure] <Jukonso> haceme tp hacia vos
[18Mar2025 19:14:52.751] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2007ms or 40 ticks behind
